mobify/platform-scaffold
# Upgrade to lighthouse 2.2.1 and parse the JSON report

## Changes
- Use Lighthouse 2.2.1

## Todos

## How to test-drive this PR
- `npm run dev` then `npm run test:pwa-preview`
- Verify that the lighthouse score, bundle size, and first interactive values are printed
